src/components/Menu/Menu.module.css

```
@@ -0,0 +1,30 @@
+/*------------------------------------*\
+    # MENU
+\*------------------------------------*/
+
+/**
+ * Menu
+ */
+.menu {
+  position: relative;
+}
+
+.menu__popover {
+  max-width: var(--eds-l-max-width);
+  margin-top: var(--eds-size-1);
+  position: absolute;
+}
+
+.menu__button {
+  background-color: var(--light-color-button-background);
+}
+
+/* // is getting overridden by a style in clickablestyle */
+.menu__button--icon {
+  color: var(--eds-theme-color-button-secondary-neutral-text);
```

Let's use the token design is using.
```suggestion
  color: var(--eds-theme-color-icon-neutral-default);
```